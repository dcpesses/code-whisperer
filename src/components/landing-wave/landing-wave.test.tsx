import {render} from '@testing-library/react';
import LandingWave from './index';

describe('LandingWave', () => {
  test('Should render as expected', () => {
    const {container} = render(<LandingWave />);
    expect(container).toMatchSnapshot();
  });
});
