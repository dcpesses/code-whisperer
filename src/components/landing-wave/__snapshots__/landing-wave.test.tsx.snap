// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LandingWave > Should render as expected 1`] = `
<div>
  <div
    class="landing-wave"
  >
    <section>
      <div
        class="wave wave1"
      />
      <div
        class="wave wave2"
      />
      <div
        class="wave wave3"
      />
      <div
        class="wave wave4"
      />
    </section>
  </div>
</div>
`;
